tests/support/tdd.js

```diff
@@ -0,0 +1,37 @@
+define([ 'intern!tdd' ], function (tdd) {
+  function catchGlobalErrors(testFn) {
+    return function () {
+      var promise = testFn.apply(this, arguments);
+
+      // testFn.length is checked ONLY to ensure we wrap functions that
+      // were written to receive a `done` function so they can be fixed
+      if (promise && promise.then || this.isAsync || testFn.length > 0) {
```

Comment:
Oh, I just copied and pasted the original thing I wrote without modifying it. This can go away.